### PR TITLE
Add a little flair to the Root cmd (Astronomer astro-cli splash screen)

### PR DESCRIPTION
**Pull request by alex-astronomer**
## Description

Adds an Astro-CLI splash screen to the root cmd.

## 🧪 Functional Testing

Purely aesthetic.

## 📸 Screenshots

Screen Shot 2022-09-28 at 10 14 41

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `make test` before taking out of draft
- [x] Ran `make lint` before taking out of draft
- [x] Added/updated applicable tests
- [x] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [x] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [x] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [x] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: cmd/root.go
Comment (andriisoldatenko) [inline cmd/root.go:84]: ❤️ Love it!

LGTM!

**Review by andriisoldatenko — APPROVED** [review]

**Review by neel-astro — APPROVED** [review]

**Comment by alex-astronomer** [comment]
Astronaut could probably be a little more centered... Going to work on getting that squared up

**Review by sunkickr — APPROVED** [review]
@alex-astronomer where you able to work on centering the astronaut. I'm going to ask others to review before we merge

**Comment by sunkickr** [comment]
@paolaperaza @jwitz

**Comment by alex-astronomer** [comment]
Had it pretty good the first time!  One space off from center :D

**Comment by neel-astro** [comment]
Are we good to merge this or is there anything pending in here?
cc @alex-astronomer @sunkickr

File: cmd/root.go
Comment (jwitz) [inline cmd/root.go:84]:
```suggestion
Welcome to the Astro CLI, the modern command line interface for data orchestration. You can use it for Astro, Astronomer Software, or local development.`,
```
One minor edit to make it clear that "Astro" is not the CLI but the product that you can use with the CLI

**Comment by kushalmalani** [comment]
@alex-astronomer We are moving away from the astronomy analogy. Hence removing the astronaut from the splash screen 😕

File: cmd/root.go
Comment (jemishp) [reply cmd/root.go]: Awesome. LGTM!

**Comment by alex-astronomer** [comment]
No worries, what's the reason for that? I thought Astronomer and Astro was like directly related to space lol not trying to be sassy just curious.

**Comment by kushalmalani** [comment]
I think its just a product decision. @sunkickr Do you remember what was the reason?

**Pull request merged**
